streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/KubernetesRetriever.scala

```diff
@@ -42,11 +42,11 @@ import scala.util.{Failure, Success, Try}
 object KubernetesRetriever extends Logger {
 
   // see org.apache.flink.client.cli.ClientOptions.CLIENT_TIMEOUT}
-  val FLINK_CLIENT_TIMEOUT_SEC = 30L
+  val FLINK_CLIENT_TIMEOUT_SEC = 60L
   // see org.apache.flink.configuration.RestOptions.AWAIT_LEADER_TIMEOUT
-  val FLINK_REST_AWAIT_TIMEOUT_SEC = 10L
+  val FLINK_REST_AWAIT_TIMEOUT_SEC = 30L
   // see org.apache.flink.configuration.RestOptions.RETRY_MAX_ATTEMPTS
-  val FLINK_REST_RETRY_MAX_ATTEMPTS = 2
+  val FLINK_REST_RETRY_MAX_ATTEMPTS = 30
```

Comment:
Why not use RestOptions.RETRY_MAX_ATTEMPTS and instead redeclare the variable  FLINK_REST_RETRY_MAX_ATTEMPTS？

Reply:
Emmm... For the streampark project, I don't know how to use FLINK's `RestOptions.RETRY_MAX_ATTEMPTS` variable.

Reply:
> Why not use RestOptions.RETRY_MAX_ATTEMPTS and instead redeclare the variable FLINK_REST_RETRY_MAX_ATTEMPTS？

cc @Al-assad

Reply:
@wolfboys https://github.com/apache/incubator-streampark/pull/3416